tests/web/test_metadata.py

```diff
@@ -56,21 +56,21 @@ def test_get_metadata(client, b):
 
 
 @pytest.mark.genesis
-def test_get_metadata_limit(client, b):
+def test_get_metadata_limit(client, b, alice):
```

Comment:
Same here, no need to fix this. The test [here](https://github.com/bigchaindb/bigchaindb/pull/2338/files#diff-d67953f9f41851852e17a20de0265318R124) covers this.